src/test/kotlin/fi/metatavu/lipsanen/functional/TaskTestIT.kt

```diff
@@ -258,6 +258,160 @@ class TaskTestIT : AbstractFunctionalTest() {
         assertEquals(taskUpdateData.endDate, foundMilestone.endDate)
     }
 
+    /*
+    Test case for cascade updating the tasks forward:
+    Source:
+    Day:    1   2   3   4   5   6
+    Task:  *t1***
+                *t2**
+                    *t3******
+                        *t4**
+
+    With expected outcome after moving t1 to 3-4
+    Day:    1   2   3   4   5   6
+    Task:           **t1*
+                        **t2*
+                        *t3******
+                        *t4******
```

Comment:
Great idea to visualize the changes in a comment!

It could maybe be even more illustrative with something like this:

```suggestion
    Test case for cascade updating the tasks forward:
    Source:
    Day:    1   2   3   4   5   6
    Task:   |t1-|
                |t2-|
                    |t3-----|
                        |t4-|

    With expected outcome after moving t1 to 3-4
    Day:    1   2   3   4   5   6
    Task:           |t1-|
                        |t2-|
                        |t3-----|
                        |t4-----|
```